server/package.json

```diff
@@ -5,13 +5,15 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "nodemon"
+    "dev": "nodemon",
+    "deploy": "sls deploy --env_src=remoteEnv --verbose --aws-s3-accelerate",
```

Comment:
tells serverless to evaluate serverless.yml file with env_src=remoteEnv so that correct configuration yaml is used
create additional yamls per environment and specify env_src=<my_env>